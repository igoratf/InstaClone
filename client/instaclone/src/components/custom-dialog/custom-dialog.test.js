import React from 'react';
import { shallow } from 'enzyme';
import CustomDialog from './custom-dialog.component';

describe('CustomDialog component', () => {
  it('should render CustomDialog component', () => {
    const mockProps = {
      open: jest.fn(),
      close: jest.fn()
    }
    expect(shallow(<CustomDialog {...mockProps}/>)).toMatchSnapshot();
  })
})