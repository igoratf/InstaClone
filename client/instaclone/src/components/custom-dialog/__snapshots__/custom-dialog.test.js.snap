// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomDialog component should render CustomDialog component 1`] = `ShallowWrapper {}`;
